README.md

```diff
@@ -28,6 +28,7 @@ Usage of netapp-exporter:
   interval: 30                                  # Collect Interval
   username: "XXX"                               # Netapp SSH Username
   password: "XXX"                               # Netapp SSH Password
+  known_hosts_file: "/root/.ssh/known_hosts"    # For SSH Secure Connection
```

Comment:
Why are u using root to connect through SSH? Cannot u use a non-root user?

Reply:
This is only for example :) Any path with known_hosts is good